sdk/scheduler/src/main/java/com/mesosphere/sdk/config/TaskEnvRouter.java

```diff
@@ -21,7 +21,7 @@
 public class TaskEnvRouter {
 
     private static final String TASKCFG_PREFIX = "TASKCFG_";
-    private static final String TASKCFG_GLOBAL_ENV_PREFIX = TASKCFG_PREFIX + "ALL_";
+    public static final String TASKCFG_GLOBAL_ENV_PREFIX = TASKCFG_PREFIX + "ALL_";
```

Comment:
I'm not sure about making this static public just to use in the HDFS scheduler. Could we not use the methods of the `TaskEnvRouter` to get done what we want in this case?

@benclarkwood I would love to hear your thoughts here.

Reply:
No method in the class provides the functionality of prepending a given env var name with the desired prefix, unless you're suggesting we write one?